clients/js/test/dasApi.test.ts

```diff
@@ -109,3 +122,540 @@ test('it can fetch the proof of a compressed asset by ID', async (t) => {
     tree_id: '6tPxkhcjcfR7rXsnGwzh8rPnkiYt2r6tDGN1TUv4T15E',
   });
 });
+
+test('it can fetch the proof of a compressed asset', async (t) => {
+  // Given a minted NFT on devnet.
+  const { umi } = t.context;
+  const assetId = publicKey('BZHZ4GX7JZ1JxngRVtHJgUAnUZQ76ffBTsNXuqTVXvg5');
```

Comment:
Nit: would be convenient if in the comments for each tree you said the extra tree properties relevant to the test.  i.e. "Given a tree with max depth 14 and canopy size X"  and
"Then we expect the following data with X proof nodes in the `proof` field for the `AssetWithProof`.

Although its fairly straightforward especially from the test names, this little extra would make it super obvious what the exact intention of each test is.